tests/integration/cli_test.go

```diff
@@ -0,0 +1,26 @@
+//go:build integration
+// +build integration
```

Comment:
@carolynvs Since this `go:build` tag is now preferred on 1.17 over `// +build`, should we replace all of the build tag with the appropriate one on `release/v1` branch?

Reply:
Now that we are on 1.18, yes I think we can drop the deprecated build tag lines.